### PR TITLE
Add step to PR template

### PR DESCRIPTION
**Pull request by kiryldz**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Fixes: < Link to related issues that will be fixed by this pull request, if they
  - [ ] Document any changes to public APIs.
  - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
  - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
+ - [ ] Port this PR to `v10.[version]` release branch if this change should be included in upcoming `v10.[version]-rc.[number]` or `v10.[version]` stable release.
```
Comment (yunikkk) [inline .github/pull_request_template.md:27]: It's only about time span when release have alread been started but not yet finished, right?

Comment (Chaoba) [reply .github/pull_request_template.md:27]: How to `port` here? Cherry-pick?

Comment (pengdev) [reply .github/pull_request_template.md:27]: what if it's a PR towards the release branch, shall we mention to port the fix to the main branch as well?(e.g change log for rc and final releases.)

Comment (kiryldz) [reply .github/pull_request_template.md:27]: @pengdev - imho it's impossible situation, if we have some branch targeted to release branch, we firstly prepare PR to `main` in any case and then port it to release branch

@Chaoba - I did not mention cherry-pick as situations may happen when cherry-pick will be impossible due to some conflicts between `main` branch and release branch. It's PR author's responsibility to select correct way here, in 90% of cases cherry-pick should be enough but sometimes PR should be open explicitly.

@yunikkk no, it's about 4-week release adoption time. When we cut `beta` for some release - this point I've added becomes valid till we release stable version. Our regular schedule is `beta` -> `rc` after 2 weeks -> stable version after 2 weeks more.

Comment (pengdev) [reply .github/pull_request_template.md:27]:
```suggestion
 - [ ] Fixes for the release branch should be merged to `main` branch first and then ported to the release branch.
   - [ ] i.e. the PR should be ported to `v10.[version]` release branch if this change should be included in upcoming `v10.[version]-rc.[number]` or `v10.[version]` stable release.
```

**Review by yunikkk — APPROVED** [review]

**Pull request merged**